test_runner/fixtures/compare_fixtures.py

```diff
@@ -333,3 +338,27 @@ def neon_with_baseline(request: FixtureRequest) -> PgCompare:
     fixture = request.getfixturevalue(request.param)
     assert isinstance(fixture, PgCompare), f"test error: fixture {fixture} is not PgCompare"
     return fixture
+
+
+@pytest.fixture(scope="function", autouse=True)
+def sync_after_each_test():
+    # some of the benchmarks are quite write-happy, and create issues to start the processes up in 10s.
+    # SYNC_AFTER_EACH_TEST is only "true" for benchmarks, which run test_runner/performance cases sequentially
+    key = "SYNC_AFTER_EACH_TEST"
+    enabled = os.environ.get(key) == "true"
+    yield
+    if not enabled:
+        log.info("not calling sync after test")
+        return
```

Comment:
This might be a bit too noisy

Reply:
yes this is just for me to help with spot checking if this gets applied how I assume. I will remove it.

Reply:
Removed.